resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -334,6 +334,8 @@ spec:
               value: "{{ .Values.gardener.autoUpdateMachineImageVersion }}"
             - name: APP_PROVISIONER_MULTI_ZONE_CLUSTER
               value: "{{ .Values.gardener.multiZoneCluster }}"
+            - name: APP_PROVISIONER_CONTROL_PLANE_FAILURE_TOLERANCE
+              value: "{{ .Values.gardener.controlPlaneFailureTolerance }}"
```

Comment:
nit: given this is configured in KEB for all requests the same, does it make sense to keep passing this explicitly from KEB in every gardener payload to provisioner? could be simpler to configure once in the provisioner and save on the payloads.

Reply:
This is not passed in every payload from KEB to provisioner, it is only passed for shoots of enterprise plans, not for trial, free and azure_lite.
provisioner doesn't have the knowledge about "SKR plans", and wouldn't know to which cluster's should it apply the configured failure tolerance configuration.